WindowsServerDocs/administration/windows-commands/sc-delete.md

```diff
@@ -43,11 +43,20 @@ Use **Add or Remove Programs** on **Control Panel** to delete DHCP, DNS, or any
 
 ## Examples
 
+### DOS Examples
```

Comment:
"DOS" is the name of an operating system that went extinct 25 years ago. I've seen people confusing Command Prompt with DOS, but an official documentation page is not written by confused people.